opentelemetry-sdk/src/trace/provider.rs

```diff
@@ -13,16 +13,14 @@ use crate::trace::{
     BatchSpanProcessor, Config, RandomIdGenerator, Sampler, SimpleSpanProcessor, SpanLimits, Tracer,
 };
 use crate::{export::trace::SpanExporter, trace::SpanProcessor};
-use crate::{InstrumentationLibrary, Resource};
+use crate::{Resource, Scope};
 use once_cell::sync::{Lazy, OnceCell};
 use opentelemetry::trace::TraceError;
 use opentelemetry::{global, trace::TraceResult};
 use std::borrow::Cow;
 use std::sync::atomic::{AtomicBool, Ordering};
 use std::sync::Arc;
 
-/// Default tracer name if empty string is provided.
-const DEFAULT_COMPONENT_NAME: &str = "rust.opentelemetry.io/sdk/tracer";
```

Comment:
Same as this https://github.com/open-telemetry/opentelemetry-rust/pull/2220/files#r1811937347